experiments/react-native-uifabric/src/components/PersonaCoin/PersonaCoin.helpers.ts

```diff
@@ -0,0 +1,97 @@
+import { PersonaSize, PersonaCoinColor, PersonaPresence } from './PersonaCoin.types';
+import { ImageURISource } from 'react-native';
+
+interface Dictionary<T> {
+  [key: string]: T;
+}
+
+const presenceIconCache: Dictionary<ImageURISource> = {

```

Comment:
Long term we need a better strategy than having these embedded images

Reply:
Yup, I'll switch to using Icon/SVG when it is available.